R/transformCounts.R

```diff
@@ -1,7 +1,7 @@
-#' Transform Abundance
+#' Transform Counts
 #'
 #' These functions applies transformation to abundance table. By using
```

Comment:
Alternative: "These functions provide a variety of options for transforming abundance data." By using ...

Reply:
That sounds better, changed